### PR TITLE
Updates to Personalized Variant for GA

### PR DESCRIPTION
**Pull request by borisrev**
# Pull Request/Issue Resolution

Closes #**BD-1696**

#### Is this change associated with a Braze feature/product release?
- [x] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @timothy-kim or @KellieHawks for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@Timothy-Kim</td>
    <td>Data Infrastructure<br>Platform Infrastructure<br>Quality Infrastructure</td>
  </tr>
  <tr>
    <td>@kelliehawks</td>
    <td>Internal Tools<br>Product Partnerships<br>SDKs<br>SMS</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Channels<br>FIX<br>Intelligence<br>Reporting<br>SMB</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Email (Composition and Infrastructure)<br>Ingestion<br>Messaging</td>
  </tr>
</table>
</details>

### CODE REVIEW

File: _docs/_user_guide/engagement_tools/testing/multivariant_testing/multivariate_analytics.md [hunk 1]
```diff
@@ -86,9 +86,9 @@ For example, the test may find that most users prefer Variant A, but users who h
...
+An individual user's recommended message is the sum of the effects of their specific recency, frequency, and tenure. Recency, frequency, and tenure are split into buckets, as illustrated in the **User Characteristics** table. The extent or width of each bucket is determined by the data for users in each individual campaign and will therefore change from campaign to campaign. 
 
-This table only summarizes which variant each bucket tends to engage with. It does not guarantee that the "strength" of any one bucket will dominate the others. Therefore a user's personalized variant depends on the sum of the effects of the three buckets they're in—one for each characteristic.
+Each bucket can have a different contribution at a different "strength" for each message variant. This table only summarizes which variant users in each bucket tends to engage with. It does not guarantee that the "strength" of any one bucket will dominate the others. Therefore a user's personalized variant depends on the sum of the effects of the three buckets they're in—one for each characteristic.
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/engagement_tools/testing/multivariant_testing/multivariate_analytics.md:91]:
```suggestion
Each bucket can have a different contribution at a different "strength" for each message variant. This table only summarizes which variant users in each bucket tend to engage with. It does not guarantee that the "strength" of any one bucket will dominate the others. Therefore a user's personalized variant depends on the sum of the effects of the three buckets they're in—one for each characteristic.
```

File: _docs/_user_guide/engagement_tools/testing/multivariant_testing/multivariate_analytics.md [hunk 2]
```diff
@@ -86,9 +86,9 @@ For example, the test may find that most users prefer Variant A, but users who h
 
 #### How personalized variants are selected
 
-An individual user's recommended message is the sum of the effects of their specific recency, frequency, and tenure. Recency, frequency, and tenure are split into buckets, as illustrated in the **User Characteristics** table. Each bucket can have a different contribution at a different "strength" for each message variant. 
+An individual user's recommended message is the sum of the effects of their specific recency, frequency, and tenure. Recency, frequency, and tenure are split into buckets, as illustrated in the **User Characteristics** table. The extent or width of each bucket is determined by the data for users in each individual campaign and will therefore change from campaign to campaign.
```
[the remark this answers is not in the document]
Comment (borisrev) [reply _docs/_user_guide/engagement_tools/testing/multivariant_testing/multivariate_analytics.md:87]: It won't always be days unfortunately. Could be hours

[the remark this answers is not in the document]
Comment (bre-fitzgerald) [reply _docs/_user_guide/engagement_tools/testing/multivariant_testing/multivariate_analytics.md:87]: Hmm okay. Time range maybe?

File: _docs/_user_guide/engagement_tools/testing/multivariant_testing/multivariate_analytics.md [hunk 3]
```diff
@@ -86,16 +84,16 @@ For example, the test may find that most users prefer Variant A, but users who h
...
+Each bucket can have a different contribution or "push" towards each message variant. The strength of the push for each bucket is determined from user responses in the initial send using logistic regression. This table only summarizes the results by displaying which variant users in each bucket tended to engage with. But, any individual user's actual personalized variant depends on the sum of the effects of the three buckets they're in—one for each characteristic.
 
 {% endtab %}
 {% tab Personalized Variant %}
 
 The **Personalized Variant** tab shows the results of the second send, where each remaining user was sent the variant they were most likely to engage with.
 
-The three cards on this page show your projected lift, overall results, and the projected results if you sent just the winning variant instead.
+The three cards on this page show your projected lift, overall results, and the projected results if you sent just the winning variant instead. Even if Personalized Variant fails to provide any lift, which can sometimes happen, the result is equivalent to sending only the winning variant, i.e. a traditional A/B test.
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/engagement_tools/testing/multivariant_testing/multivariate_analytics.md:96]:
```suggestion
The three cards on this page show your projected lift, overall results, and the projected results if you sent just the winning variant instead. Even if there's no lift, which can sometimes happen, the result is the same as sending only the winning variant (a traditional A/B test). 
```

**Review by bre-fitzgerald — CHANGES_REQUESTED** [review]

**Review by bre-fitzgerald — APPROVED** [review]

**Pull request merged**